pkg/virt-api/webhooks/validating-webhook/admitters/vms-admitter_test.go

```diff
@@ -2051,6 +2053,19 @@ var _ = Describe("Validating VM Admitter", func() {
 				}),
 			)
 		})
+
+		Context("Update volume strategy", func() {
+			It("should reject the VM creation if the feature gate isn't enabled", func() {
```

Comment:
Makes sense to add a positive test case with FG enabled?

Reply:
Yes, I just added, thanks!